### PR TITLE
make modoboa-admin deploy work with python 3.4

### PR DESCRIPTION
**Pull request by arthru**
Description of the issue/feature this PR addresses:
The command `modoboa-admin.py deploy --collectstatic --dburl default:postgres://localhost/modoboa_test --domain localhost --admin-username admin modoboa_test` failed with a lot of errrors when launched with python 3.4. The PR aims to  fix that.

Current behavior before PR:
Errors when launching ; return code != 0

Desired behavior after PR is merged:
No errors ; return code == 0

### CODE REVIEW

File: modoboa/admin/management/commands/subcommands/_repair.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.utils.encoding import smart_str
```
Comment (tonioo) [inline modoboa/admin/management/commands/subcommands/_repair.py]: smart_text ?

Comment (arthru) [inline modoboa/admin/management/commands/subcommands/_repair.py]: smart_str returns str in both python 2 and 3 whereas smart_text returns unicode in python 2 and str in python3. Sometimes, we need smart_str when the API keeps using str in both python 2 and 3....

**Comment by arthru** [comment]
So, the tests are ok in both python 2 and python 3...

I am going to install a live instance on this branch to validate that everythings work fine.

**Comment by arthru** [comment]
BTW, I don't know why there is a problem with codecov/project... and i can't access the "Details" page...

**Pull request merged**